text.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main>
+        <header class="header">
+            <div class="logo">Jo</div>
+            <div></div>
+            <div class="menu">
+                <a href="#" class="nav">
+                    <div ></div>
+                    <div ></div>
+                    <div ></div>
+                </a>
+            </div>
+        </header>
+        <section class="headline">
+            <div></div>
+            <div class="greet">I'm Jonathan Glad to see you!</div> 
```

Comment:
- [ ] Kindly ensure that the font size, and font weight of the headline text match the Figma design you chose. You can check out the Figma design specification for the exact properties used👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/191637166-6826c67f-4f55-4f01-a4ac-16126a6f80ca.png))  |![image](https://user-images.githubusercontent.com/63473584/191637203-a7bc2b38-dfad-4286-848d-288b812b1ee3.png)  |